src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplCaseInvalidateTest.java

```diff
@@ -20,6 +21,7 @@
 import uk.gov.ons.ctp.common.event.EventPublisher.EventType;
 import uk.gov.ons.ctp.common.event.EventPublisher.Source;
 import uk.gov.ons.ctp.common.event.model.AddressNotValid;
+import uk.gov.ons.ctp.integration.caseapiclient.caseservice.model.CaseContainerDTO;
```

Comment:
Since we already have the static import `Mockito.when` , then all occurrences of `Mockito.when` can be replaced with just `when` .  One way of doing this is to put the cursor on the `when` and do the key sequence CMD-SHIFT-M

Reply:
interesting info - done